opencti-platform/opencti-front/src/private/components/data/drawers/PromoteDrawer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import Typography from '@mui/material/Typography';
+import Alert from '@mui/material/Alert';
+import Button from '@mui/material/Button';
+import Drawer from '@components/common/drawer/Drawer';
+import { useFormatter } from '../../../../components/i18n';
+import type { Theme } from '../../../../components/Theme';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  buttons: {
+    marginTop: 20,
+    textAlign: 'right',
+  },
+  button: {
+    marginLeft: theme.spacing(2),
+  },
+  container: {
+    padding: '10px 20px 20px 20px',
+  },
+}));
+
+type DrawerContainerPropsType = {
+  isOpen: boolean;
+  observablesFiltered: boolean;
+  onClose: () => void;
+  onSubmit: () => void;
+  isIndicator: boolean;
+};
+
+const PromoteDrawer = ({ isOpen, onClose, observablesFiltered, onSubmit, isIndicator }: DrawerContainerPropsType) => {
```

Comment:
I don't really understand the parameters observablesFiltered and isIndicator. What values can they have ? Can we have observablesFiltered and isIndicator true ? so we have observables and indicators to promote at the same time ?